recipes/libguarded/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(main main.cpp)
+target_link_libraries(main CONAN_PKG::libguarded)
+set_property(TARGET main PROPERTY CXX_STANDARD 11)
```

```suggestion
set_property(TARGET main PROPERTY CXX_STANDARD 14)
```

It's c++14 according the project page

---

According to the [documentation](https://www.copperspice.com/docs/cs_libguarded/lib-api.html), part of the library is C++11, but you need C++14 to be able to use all of it. As the test_package shows, it is perfectly fine to use the core feature of the library with C++11.

Not sure how to reflect that on the project tags

---

Agreed!